### PR TITLE
Adding 3 games to ForceMax60FPS

### PR DESCRIPTION
**Pull request by Panderner**
See #13503 this PR improves performance for Low-end Devices without underclocking the CPU.

Unfortunately when I enable ForceMax60FPS for Open Season and Killzone but it won't improve performance

### CODE REVIEW

File: assets/compat.ini [hunk 1]
```diff
@@ -804,6 +804,17 @@ ULUS10132 = true
 # Homebrew Kurok (we don't normally add homebrew here but it has a distinctive GameID)
 KURO00767 = true
 
+# Scooby-Doo! Who's Watching Who?
+ULUS10168 = true
+ULES00571 = true
+
+# Tron Evolution
+ULUS10548 = true
+ULES01494 = true
```
Comment (hrydgard) [inline assets/compat.ini:813]: There's also ULES01495, see https://report.ppsspp.org/games?name=tron and hover the game names

**Comment by Gamemulatorer** [comment]
Can you remove duplicate id for god of war games.
IMG_20210921_071928

Kindly added aslo this game id of Kurohyou 2: Ryu ga Gotoku Ashura Hen (Japan)
NPJH-50562
NPJH-50333
As I mentioned here https://github.com/hrydgard/ppsspp/issues/5703#issuecomment-922558858

**Comment by hrydgard** [comment]
@Gamemulatorer Thanks, will take care of it.

**Comment by Panderner** [comment]
Umm any performance difference for enabling ForceMax60FPS for Kurohyou 2?

**Comment by hrydgard** [comment]
Since he requested it, I assume it doesn't hurt, at least.

**Pull request merged**